telegram/ext/_updater.py

```diff
@@ -568,6 +568,11 @@ async def _bootstrap(
         ip_address: str = None,
         max_connections: int = 40,
     ) -> None:
+        """Entry point for handling webhooks. :meth:`start_polling` calls this to delete any
+        present webhook. :meth:`start_webhook` calls this to set a webhook using
+        :meth:`telegram.Bot.set_webhook. If there are unsuccessful attempts, it will be retried as
+        specified by :paramref:`max_retries`.
+        """
```

Comment:
```suggestion
        """Prepares the setup for fetching updates: delete or set the webhook and drop pending updates if appropriate. If there are unsuccessful attempts, this will retry as
        specified by :paramref:`max_retries`.
        """
```